### PR TITLE
Patch 1

### PR DESCRIPTION
**Pull request by mclr47**
<!DOCTYPE html>
<html>
    <head>
        <title>Profile Page</title>
        <meta charset="utf-8">
       
        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.0/css/bootstrap.min.css">
        <link rel="stylesheet" type="text/css" href="group9Sprint2.css">
   <link crossorigin="anonymous" media="all" integrity="sha512-dDnCHJtEv2Hu49bx+DrWxu249TjieZ9um/g3uHqaXDSd2eNDLfSooD46uiGU3IJXiaMNJ6W0IoCcTjq6gTh8QA==" rel="stylesheet" href="https://github.githubassets.com/assets/github-3d951771a4ed5753620b461a24fd4154.css">
<link rel="fluid-icon" href="https://github.com/fluidicon.png" title="GitHub">
        
        <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>

        
        <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.0/js/bootstrap.min.js"></script>
        <script src="https://www.chartjs.org/dist/2.8.0/Chart.min.js"></script>
        <script src="repogroup.js"></script>
    </head>
    <style>
        h1{
            padding-left: 50px;
        }
        #container{
            background-color: black;
        }
        .btn{
            background-color: darkgrey;
            color: floralwhite;
            border-color: black;
        }
    </style>
    <body onload="getRepoGroupInfo()" id="bodyId">
       
<nav class="navbar navbar-default">
    <div class="container-fluid" id="container">

    <div class="collapse navbar-collapse">
      <ul class="nav navbar-nav">
        <li class="buttons">
            <div class="navbar-form">
                <div class="btn-group">
                    <a class="btn b btn-default"  href="#home">Home</a>
                    <a class="btn b btn-default"  href="#about">About</a>
                    <a class="btn b btn-default"  href="#contact">Contact</a>
                    <a class="btn b btn-default"  href="#profile">Profile</a>
                    <a class="btn b btn-default dropdown-toggle" data-toggle="dropdown" href="#">Settings<span class="caret"></span></a>
                  <ul class="dropdown-menu">
                    <li><a href="#sort">Sort Data</a></li>
                    <li><a href="#logout">Logout</a></li>
                  </ul>
                    

                </div>
                
            </div>
        </li> 
      </ul>
     </div>
    </div>
    
           
</nav>
        
        <h1>Welcome Owner Of The GraphQL Repo Group!</h1>
        <div class="dropdown">
  <button class="dropbtn border-button ">GitHub Investigated Features</button>
  <div class="dropdown-content">
    <a href="#" class="border-button">repo_group_id</a>
    <a href="#" class="border-button">rg_name</a>
    <a href="#" class="border-button">rg_website</a>
    <a href="#" class="border-button">rg_recache</a>
    <a href="#" class="border-button">rg_last_modified</a>
    <a href="#" class="border-button">rg_type</a>
    <a href="#" class="border-button">tool_source</a>
    <a href="#" class="border-button">tool_version</a>
    <a href="#" class="border-button">data_source</a>  
    <a href="#" class="border-button">data_collection_date</a>  
      
      
  </div>

</div>  
        <script>
//            functions in javascript
//            routing
            
            import { HomeComponent } from './home/home.component';
            import { AboutComponent } from './about/about.component';
            import { PrivacyComponent } from './privacy/privacy.component';
            import { TermsComponent } from './terms/terms.component';
            
            const routes: Routes = [
              { path: 'home', component: HomeComponent },
              { path: 'about', component: AboutComponent },
              { path: 'contact', component: ContactComponent },
              { path: 'profile', component: ProfileComponent },
              { path: '', redirectTo: '/home', pathMatch: 'full' }
            ];
        </script>
        
        <div class="row">
            <div class="col-sm-9" id="sectionBody"></div>
        </div>
        <div class="row">
            <div class="col-sm-9" id="sectionBody2"></div>
        </div>
        <div class="row">
            <div class="col-sm-9" id="sectionBody2"></div>
        </div>
    </body>
</html>

### CODE REVIEW

**Comment by mclr47** [comment]
/*//group9Sprit2.css*/
.b {
 
  cursor: url(https://github.com/fluidicon.png=20x20),auto,width="100" height="100";
 -webkit-zoom-in
-webkit-zoom-out
-webkit-zoom-grab
-webkit-zoom-grabbing
}
#img { width: 50px;
      height: 50px;
    background-size: contain;
    float:right;
    
}
img:hover{width: 40px;
          height:40px;}


#bodyId
{text-align: center;}
/*
.b:nth-last-child(even){
    background-clip: border-box;
    border-left-color: aqua;
    background-color: yellow;
    //border-radius:50%;
} 
*/
.b{width:auto;}

.btn-group:nth-of-type(even){
    border:radius 40%;
    
}
.b:nth-child(odd){
    background-clip: border-box;
    border-left-color: #999890;
    color: #aaaaff;
    background-color: 71C3CF;
/*    //border-radius:50%;*/
} 
.b:nth-child(even):hover{
    border-radius: 50%;
    background-color: darkgreen;
    color:darkblue;
    font-weight: 700;
}
.b:nth-child(odd):hover{
    border-radius: 50%;
    background-color:orange;
    color:black;
    font-weight: 700;
}

/**************************************/

.dropbtn {
  background-color: #5b5b6e;
  color: white;
  padding: 16px;
  font-size: 16px;
  border: none;
    min-width: 160px;
}
.dropbtn :hover{
    background-color: darksalmon;
}

.dropdown {
  position: relative;
  display: inline-block;
  text-align: center; 
  background-color: bisque;
    border-bottom-style: groove;
    min-width: 160px;
    
}

.dropdown-content {
  display: none;
  position: absolute;
  background-color: #f1f1f1;
  min-width: 160px;
  box-shadow: 0px 8px 16px 0px rgba(0,0,0,0.2);
  z-index: 1;
    text-align: center; 
    position: relative;
    overflow: scroll;
}

.dropdown-content a {
  color: black;
  padding: 12px 16px;
  text-decoration: none;
  display: block;
    text-align: center;
}

.dropdown-content a:nth-child(even) {
  color: black;
  padding: 12px 16px;
  text-decoration: none;
  display: block;
    text-align: center;
    background-color: aqua;
    font-size: 20px;
    font-weight: 500;
    font-family: sans-serif;
}

.dropdown-content a:nth-child(odd) {
  color: black;
  padding: 12px 16px;
  text-decoration: none;
  display: block;
    text-align: center;
    background-color: #dfdd97;
    font-size: 20px;
    font-weight: 500;
    
}

.dropdown-content a:hover {background-color: #ddd;
                            font-stretch: extra-expanded;
                            font-kerning: normal;
                            font-weight: 900; 
}

.dropdown:hover .dropdown-content {display: block;
  text-align: center;}

.dropdown:hover .dropbtn {background-color: orange;}

 .border-button {
  border: solid 5px #510085;
  transition: border-width 0.01s linear;
}

.border-button:hover {  border:solid grey 10px;
                        border-width: 10px;
                        border-radius: 20%;}
